.goreleaser.yml

```diff
@@ -54,3 +53,31 @@ dockers:
     extra_files:
       - etc/chart_schema.yaml
       - etc/lintconf.yaml
+  - skip_push: false
+    dockerfile: Dockerfile
+    image_templates:
+      - quay.io/helmpack/chart-testing:{{ .Tag }}-arm64
+      - quay.io/helmpack/chart-testing:latest-arm64
+    build_flag_templates:
+      - --platform=linux/arm64
+      - --label=org.opencontainers.image.version={{ .Version }}
+      - --label=org.opencontainers.image.revision={{ .Commit }}
+      - --label=org.opencontainers.image.title={{ .ProjectName }}
+      - --label=org.opencontainers.image.created={{ .Date }}
+      - --label=org.opencontainers.image.description=ct - The chart testing tool
+      - --label=org.opencontainers.image.vendor=Helm
+      - --label=org.opencontainers.image.licenses=Apache-2.0
+      - --label=org.opencontainers.image.source=https://github.com/helm/chart-testing
+      - --label=org.opencontainers.image.authors=The Helm Authors
+    extra_files:
+      - etc/chart_schema.yaml
+      - etc/lintconf.yaml
+docker_manifests:
+- name_template: quay.io/helmpack/chart-testing:latest
+  image_templates:
+  - quay.io/tauerbec/chart-testing:latest-amd64
```

Comment:
shouldn't these be `quay.io/helmpack/chart-testing` ?

Reply:
@paulczar Fixed.